tests/validate-perfetto-proto.py

```diff
@@ -2,18 +2,22 @@
 
 import sys
```

Comment:
```suggestion
import sys
import os
```
Required for the error checking.